portal/server/types/db-models/banks.ts

```diff
@@ -0,0 +1,21 @@
+import { WithId } from 'mongodb';
+
+type ReportPeriodSchedule = {
+  startMonth: number;
+  endMonth: number;
+}
```

Comment:
There should be a semi-colon here (prettier will add it)

Reply:
Oops, this might be missing on feature branch too